pkg/aws/policies.go

```diff
@@ -678,3 +678,92 @@ func getPolicyDocument(policyDocument *string) (PolicyDocument, error) {
 	}
 	return data, nil
 }
+
+func (c *awsClient) DeleteOperatorRole(role string) error {
+	r := aws.String(role)
+	err := c.deleteOperatorRolePolicies(r)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			case iam.ErrCodeNoSuchEntityException:
+				fmt.Printf("Policies does not exists for role '%s'",
+					role)
+			}
+		}
+		return err
+	}
+	_, err = c.iamClient.DeleteRole(&iam.DeleteRoleInput{RoleName: r})
+	if err != nil {
+		if err != nil {
+			if aerr, ok := err.(awserr.Error); ok {
+				switch aerr.Code() {
+				case iam.ErrCodeNoSuchEntityException:
+					return fmt.Errorf("operator role '%s' does not exists.skipping",
+						role)
+				}
+			}
+			return err
+		}
+	}
+	return nil
+}
+
+func (c *awsClient) deleteOperatorRolePolicies(role *string) error {
```

Comment:
This should be renamed to `detachOperatorRolePolicies` to avoid confusion, as in the future we may want a way of deleting those too